Packs/Core/Scripts/SearchIndicatorInEvents/README.md

```diff
@@ -0,0 +1,32 @@
+Searches for a specific indicator in the tenant's events and logs data and extract the logs which the indicator appears in.
```

Comment:
Recommended:
Searches for a specific indicator in the tenant's event and log data, and extracts the logs the indicator appears in.